File: .github/ai-ley/instructions/languages/examples/java/example-2.java
// ✅ Good: Use camelCase for variables and methods
String userName = "john_doe";
int orderCount = 42;
boolean isActive = true;

public void processOrder(Order order) { }
public Optional<User> findUserById(long id) { }

// ✅ Good: Use PascalCase for classes and interfaces
public class UserService { }
public interface OrderRepository { }
public enum UserStatus { ACTIVE, INACTIVE, SUSPENDED }

// ✅ Good: Use UPPER_SNAKE_CASE for constants
public static final int MAX_RETRY_COUNT = 3;
public static final String DEFAULT_ENCODING = "UTF-8";
private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);

// ✅ Good: Use descriptive names
List<User> activeUsers = findActiveUsers();
Map<String, Integer> userCountByDepartment = calculateUserStats();

File: .github/ai-ley/instructions/languages/examples/swift/example-2.swift
// ✅ Good: Use camelCase for variables and functions
let userName = "john_doe"
let userAge = 25

func calculateTotalPrice(for items: [Item]) -> Decimal {
    return items.reduce(0) { $0 + $1.price }
}

// ✅ Good: Use PascalCase for types
struct UserProfile {
    let name: String
    let email: String
}

enum NetworkResult<T> {
    case success(T)
    case failure(Error)
}

// ✅ Good: Use descriptive names
let isUserAuthenticated = true
let hasValidEmail = user.email.contains("@")
func findUser(by email: String) -> User?